String/CountCharacters.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @function countCharacters
+ * @description Given a string, count the number of each character.
+ * @param {String} str - The input string
+ * @return {Object} - Object with characters and number of times
+ * @example countCharacters("hello") => {h: 1, e: 1, l: 2, o: 1}
+ */
+
+const countCharacters = (str) => {
+  const specialChars = /[`!@#$%^&*()_+\-=[\]{};':"\\|,.<>?~]/
```

Comment:
Why all this regex special treatment and validation? Just ignore characters outside of `a-zA-Z`. But why limit yourself to letters in the first place? This function should be called `countLetters`.

Reply:
I have uploaded changes. This algorithm only counts letters ;)